packages/suite/src/support/messages.ts

```diff
@@ -84,10 +84,31 @@ export default defineMessages({
             'A watch-only account is a public address you’ve imported into your wallet, allowing the wallet to watch for outputs but not spend them.',
         id: 'TR_ACCOUNT_IMPORTED_ANNOUNCEMENT',
     },
+    TR_HIDE_SCAM_TRANSACTIONS_TOOLTIP: {
+        defaultMessage: 'Want a cleaner view? Try hiding suspicious transactions.',
+        id: 'TR_HIDE_SCAM_TRANSACTIONS_TOOLTIP',
+    },
+    TR_SHOW_SUSPICIOUS_TRANSACTIONS: {
+        defaultMessage: 'Show suspicious transactions',
+        id: 'TR_SHOW_SUSPICIOUS_TRANSACTIONS',
+    },
+    TR_HIDE_SUSPICIOUS_TRANSACTIONS: {
+        defaultMessage: 'Hide suspicious transactions',
+        id: 'TR_HIDE_SUSPICIOUS_TRANSACTIONS',
+    },
+    TR_HIDE_SUSPICIOUS_TRANSACTIONS_DESCRIPTION: {
+        defaultMessage: 'Crypto moves fast. Our filters might not be always perfect.',
+        id: 'TR_HIDE_SUSPICIOUS_TRANSACTIONS_DESCRIPTION',
+    },
     TR_ACCOUNT_IS_EMPTY_TITLE: {
         defaultMessage: 'No transactions',
         id: 'TR_ACCOUNT_IS_EMPTY_TITLE',
     },
+    TR_NO_VISIBLE_TRANSACTIONS: {
```

Comment:
how to see this?

Reply:
Well, you gotta have an accout with that. I guess a simplest way is simply to comment out hte code to displaay it.